proto/zitadel/member.proto

```diff
@@ -143,3 +143,11 @@ message UserIDQuery {
         }
     ];
 }
+
+enum MemberFieldColumnName {
```

Comment:
Please comment on what fields to be considered here, for now i have added these field